spinalcordtoolbox/scripts/sct_smooth_spinalcord.py

```diff
@@ -133,6 +137,10 @@ def main(args=None):
         sigma = arguments.smooth
     remove_temp_files = arguments.r
     verbose = int(arguments.v)
+    if arguments.o is not None:
+        fname_out = arguments.o
+    else:
+        fname_out = extract_fname(fname_anat)[1] + '_smooth.nii.gz'
```

Comment:
please make sure to not change the default behaviour-- in this case if input is NII the output will be NIIGZ (whereas the current default is NII)

Reply:
Ah ! Sorry I'll change it.